src/Symfony/Component/Console/Command/Command.php

```diff
@@ -436,13 +447,13 @@ public function setProcessTitle($title)
     }
 
     /**
-     * Returns the command name.
+     * Returns the process title of the command.
      *
      * @return string The command name
      */
-    public function getName()
+    public function getProcessTitle()
```

Comment:
please don't move methods around. It does not provide any value and makes the history harder to use due to meaningless changes